recipes/xege/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class XegeConan(ConanFile):
+    name = "xege"
+    version = "20.08"
+    license = "LGPLv2.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://xege.org/"
+    description = "Easy Graphics Engine, a lite graphics library in Windows"
+    topics = ("ege", "graphics", "gui")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+    generators = "cmake"
+
+    def configure(self):
+        if self.settings.os != "Windows":
+            raise ConanInvalidConfiguration(
+                "This library is only compatible for Windows")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, "source_subfolder")
+        # This small hack might be useful to guarantee proper /MT /MD linkage
+        # in MSVC if the packaged project doesn't have variables to set it
+        # properly
+#         tools.replace_in_file("hello/CMakeLists.txt", "PROJECT(HelloWorld)",
+#                               '''PROJECT(HelloWorld)
+# include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+# conan_basic_setup()''')
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure(source_folder="source_subfolder")
```

Comment:
```suggestion
        cmake.configure()
```

a CMake wrapper calling `conan_basic_setup()` must be added